test/unit/simulation_run_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require 'mocha'
+
+class SimulationRunTest < MiniTest::Test
+
+  def test_proper_types_in_input_parameters
```

Comment:
@dkrol - w wielu miejscach w dokumentacji kodu używane jest słowo "proper" - ja myślę, że dla lepszego opisu intencji testu lepiej pisać coś konkretnego, np. w komentarzu nad tą metodą testującą napisać: "SimulationRun#input_paramters should return Hash value of input_parameter attribute if it's present and values of result Hash should have values of the same types as input_parameter attribute"

Myślę, że pisanie takich zdań ze specyfikacjami przed pisaniem właściwego testu pozwala określić dobrze wymagania i jest zgodne z podejściem BDD.